components/grid/editing/popup.md

```diff
@@ -188,6 +198,10 @@ The Popup editing mode supports [validation]({%slug common-features/input-valida
 
 The Grid exposes options to customize the edit popup and its form. Define the desired configuration in the `GridPopupEditSettings` and `GridPopupEditFormSettings` tags under the `GridSettings` tag.
 
+### Editability of Hidden Columns
+
+Staring from version 7.0, the Grid allows users to edit [hidden columns]({%slug grid-columns-visible%}) by default. To disable editing of a hidden column, set `Editable="false"` to the respective `<GridColumn>` tag.
```

Comment:
- Use "[starting with](https://docs.telerik.com/style-guide/vocabulary#starting-with)" when referencing a specific version.
- This comment applies to the TreeList as well.
```suggestion
Starting with version 7.0, the Grid allows users to edit [hidden columns]({%slug grid-columns-visible%}) by default. To disable editing of a hidden column, set `Editable="false"` to the respective `<GridColumn>` tag.
```